### PR TITLE
feat(logger): improve logging

### LINKED ISSUE #26: Write tests for the Logger parts
Those 4 files has no corresponding tests:

- [x] logger.js
- [x] logger-coralogix.js
- [x] logger-papertrail.js
- [x] syslog-stream.js

Considering the complexity of the embedded MultiLogger(s) structure, those tests are mandatory to understand how this is supposed to work (what kind of logger can you provide to `#wrap` ?!) and make sure we do not break something when changing the current code.

I tried to write some but I finally gave up because I am really not sure how this is supposed to work.

### LINKED ISSUE #31: use helix-log fields to avoid code duplication
bot the coralogix and papertrail logger use the same code to provide the `ow` log fields.
this should probably be done in a higher level logger.

### PR DESCRIPTION
**Pull request by tripodsan**
- create openwhisk logger with dedicated fields
- add tests

fixes #31, fixes #26

### CODE REVIEW

File: src/logger.js [hunk 1]
```diff
@@ -119,19 +112,26 @@ function addLogger(parent, name, createFN, cfg, params) {
...
 function setupHelixLogger(params, logger = rootLogger) {
-  // only overwrite the default console logger if there is one
-  if (logger.loggers.has('default')) {
-    // replace default logger with our own console logger
-    logger.loggers.set('default', openhwiskConsoleLogger);
-  }
+  // add openwhisklogger to helix-log logger
+  if (!logger.loggers.has('openwhisk')) {
+    const owLogger = new OpenWhiskLogger({});
+    logger.loggers.set('openwhisk', owLogger);
```
Comment (kptdobe) [inline src/logger.js]:
```suggestion
    logger.loggers.set('OpenWhiskLogger', owLogger);
```

Minor detail but other logger names are camel cased: `CoralogixLogger` and `PapertraiLogger`

File: test/logger.test.js [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+      PAPERTRAIL_TLS: 'false',
+      PAPERTRAIL_LOG_LEVEL: 'info',
+    }, myRootLogger);
+
+    log.info({ myId: 42 }, 'Hello, world');
+
+    await srv;
+
+    assert.equal(reqs.length, 2);
+    assert.ok(reqs[0].endsWith('test-my-action-name[1]:test-my-activati INFO Hello, world'));
```
Comment (kptdobe) [inline test/logger.test.js]: Why `test-my-activati` is trimmed here ?

Comment (tripodsan) [reply test/logger.test.js]: I can't remember exactly. but I thought that papertrail had a limit on the size of the `procid` field of the message.

File: src/logger.js (hunk 1 above)
Comment (tripodsan) [reply src/logger.js]: ok. updated.

**Pull request merged**